WordPress/Resources/bg.lproj/Localizable.strings

```diff
@@ -5198,7 +5232,8 @@ translators: Block name. %s: The localized block name */
 /* All Time Stats 'Posts' label
    Noun. Title. Links to the blog's Posts screen.
    Stats Posts Label
-   Title of a Reader tab showing Posts matching a user's search query */
+   Title of a Reader tab showing Posts matching a user's search query
+   Title of the screen showing the list of posts for a blog. */
```

Comment:
The number of different contexts for this same copy is a bit scary, localization-wize 😅  (but we don't have a solution for that yet anyway, so 🤷 )